src/logger.js

```diff
@@ -78,14 +79,21 @@ class OpenWhiskLogger extends MultiLogger {
     super(logger, {
       ...opts,
       filter: (fields) => {
+        const ret = {
+          ...fields,
+        };
+
         const ow = CLS_NAMESPACE.get(LOGGER_OW_FIELDS_NAME);
         if (ow) {
-          return {
-            ow,
-            ...fields,
-          };
+          ret.ow = ow;
+        }
+
+        const req = CLS_NAMESPACE.get(LOGGER_REQUEST_FIELDS_NAME);
+        if (req) {
+          ret.request = req;
```

Comment:
hmm.. I would expect `request` to be what the action receives. what about `cdn-request` ? or at least `request.cnd-url` ?  or `cnd.url` ?

Reply:
Finding the name was the hardest part of the job :) But I agree, I am really bad for finding names. I am fine with `cdn.url`.